lib/absinthe/subscription/local.ex

```diff
@@ -46,7 +46,7 @@ defmodule Absinthe.Subscription.Local do
     do_get_docs(pubsub, field, topic_fun.(mutation_result))
```

Comment:
do you need to `to_string` this or is this for mutation publishes?

Reply:
It happens on line 49.

Reply:
oh wait no I'm dumb, hold that thought

Reply:
fixed via 7b059d0